unbxdAutosuggest.js

```diff
@@ -1205,13 +1220,14 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 		}
 		, processPopularProducts: function (doc) {
 			o = {
-				autosuggest: doc.autosuggest
-				, highlighted: this.highlightStr(doc.autosuggest)
+				autosuggest: this.options.popularProducts.title in doc ? doc[this.options.popularProducts.title] : doc.autosuggest
```

Comment:
It cannot be ` this.options.popularProducts.title in doc`, It s no different from above code.

`this.options.popularProducts.title?doc[this.options.popularProducts.title] : doc.autosuggest`

Reply:
since title is defaulted to autosuggest, doc[this.options.popularProducts.title] will suffice